paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/manage/ManageNavigator.kt

```diff
@@ -154,7 +155,9 @@ internal class ManageNavigator private constructor(
     sealed class Action {
         object Back : Action()
 
-        object Close : Action()
+        // Only pass in updatedSelection if selection changed and not selection properties updated
+        // Passing in updatedSelection is for purposes of rowSelectionCallback
+        data class Close(val updatedSelection: PaymentSelection.Saved? = null) : Action()
```

Comment:
I think this is overly complicating the implementation. The callback shouldn't be called in this case, right?